Makefile

```diff
@@ -355,3 +356,44 @@ install: _output/local/bin
           done ; \
         fi
 endif
+
+define RELEASE_HELP_INFO
+# release components.
+#
+# Args:
+#   WHAT: Component names to be release. support: kubeedge/edgesite/keadm
+#         If not specified, "everything" will be build and release.
+#
+# VERSION: release version
+#        If not specified, default use GIT commit info
+#
+#
+# Example:
+#   make release
+#   make release HELP=y
+#   make release WHAT=kubeedge/edgesite/keadm
+#   make release WHAT=kubeedge VERSION=v1.9.0
+#   make release WHAT=kubeedge VERSION=v1.9.0 GOARM=GOARM7/GOARM8
+#
+endef
+.PHONY: release
+ifeq ($(HELP),y)
+release:
+	@echo "$$RELEASE_HELP_INFO"
+else ifeq ($(WHAT),edgesite)
+release:
+	KUBEEDGE_OUTPUT_SUBPATH=$(OUT_DIR) hack/make-rules/build.sh edgesite-server edgesite-agent
+	hack/make-rules/release.sh "edgesite" $(VERSION) $(GOARM)
+else ifeq ($(WHAT),keadm)
+release:
+	KUBEEDGE_OUTPUT_SUBPATH=$(OUT_DIR) hack/make-rules/build.sh keadm
```

Comment:
I integrate with `make all` cmd before make release, but I'm confused about how the binary version get v1.9.0 but not 1.9.0-54+14e4f891a66a05, So if `cloudcore --version` can get the correct version `v1.9.0`, then the release version will also be the same with binary version `v1.9.0`